eman2/convert/convert.py

```diff
@@ -152,6 +170,79 @@ def readCoordinates(mic, fileName, coordsSet, invertY=False):
                 coord.setMicrograph(mic)
                 coordsSet.append(coord)
 
+def readCoordinate3D(box, inputTomo):
+    from pwem import Domain
+    Coordinate3D = Domain.importFromPlugin("tomo.objects", "Coordinate3D", errorMsg=TOMO_NEEDED_MSG)
+    x, y, z = box[:3]
+    coord = Coordinate3D()
+    coord.setPosition(x, y, z)
+    coord.setVolume(inputTomo)
+    return coord
+
+def writeSetOfSubTomograms(subtomogramSet, path, **kwargs):
+    """ Convert the imgSet particles to .hdf files as expected by Eman.
+        This function should be called from a current dir where
+        the images in the set are available.
+        """
+    ext = pwutils.getExt(subtomogramSet.getFirstItem().getFileName())[1:]
+    if ext == 'hdf':
+        # create links if input has hdf format
+        for fn in subtomogramSet.getFiles():
+            newFn = pwutils.removeBaseExt(fn).split('__ctf')[0] + '.hdf'
+            newFn = pwutils.join(path, newFn)
+            pwutils.createLink(fn, newFn)
+            print("   %s -> %s" % (fn, newFn))
+    else:
+        from tomo.objects import Coordinate3D
```

Comment:
unused import

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/d112aa7cce21b8fe8ff3b78f8d495bf59eef37df